.github/workflows/sandbox.yml

```diff
@@ -25,7 +26,7 @@ jobs:
           build_extra_args: "--target=default --compress=true"
           context: ./
           dockerfile: docker/sandbox/Dockerfile
-          push_image_and_stages: ${{ github.event_name == 'release' }}
+          push_image_and_stages: ${{ github.event_name != 'pull_request' }}
```

Comment:
Any reason for this change

Reply:
Made changes in GitHub workflow, Now we will publish sandbox image in each release....Now we have event `push` and `pull_request`